### DISCUSSION TITLE
Allow generic data structures as inputs to the Model Cards

### DISCUSSION THREAD

**Discussion by hanneshapke** [tensorflow/model-card-toolkit#231] [Ideas]
Allow the rendering of Model Cards based on generic, non-MLMD based information in conjunction to the existing MLMD inputs.

This feature could increase the adoption by users who aren't running MLMD in production.

**Comment by codesue** [comment]
Do you have an examples in mind of what this interface might look like?

**Comment by codesue** [comment]
The [Scikit-Learn](https://www.tensorflow.org/responsible_ai/model_card_toolkit/examples/Scikit_Learn_Model_Card_Toolkit_Demo) and [Standalone](https://www.tensorflow.org/responsible_ai/model_card_toolkit/examples/Standalone_Model_Card_Toolkit_Demo) tutorials demonstrate how to use model cards with non-MLMD based information. I'm closing this discussion as resolved, but please feel free to reopen it.
